src/helm/benchmark/run_expander.py

```diff
@@ -435,6 +441,19 @@ class NumOutputsRunExpander(ReplaceValueRunExpander):
     }
 
 
+class NumTrialRunExpander(ReplaceValueRunExpander):
+    """For getting different generations for the same requests."""
+
+    name = "num_trials"
+    values_dict = {
```

Comment:
File an issue / add a comment to clean this up?

Reply:
I removed the numbers and added `heim_efficiency`.